docs/source/en/model_doc/whisper.md

```diff
@@ -90,6 +90,8 @@ As an example, the following codesnippet enables SDPA and `torch.compile` for up
 >>> # Load the Whisper model with SDPA attention
 >>> processor = WhisperProcessor.from_pretrained("openai/whisper-tiny.en")
 >>> model = WhisperForConditionalGeneration.from_pretrained("openai/whisper-tiny.en", attn_implementation="sdpa")
+>>> device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+>>> model = model.to(device)
```

Comment:
If the model is moved to `device` then we'll need to do the same thing with the inputs